src/Module/Manifest.php

```diff
@@ -44,7 +44,7 @@ public static function rawContent(array $parameters = [])
 			'icons'       => [
 				[
```

Comment:
I guess you should add the support for the 512pixel icons as well.

Reply:
My thought was to not add more than what's required. But you're right, adding the recommended one as well should do no harm.